### PR TITLE
(core) added region / namespace filter in project dashboard view

### PR DESCRIPTION
**Pull request by danielpeach**
region_filter
@anotherchrisberry PTAL

### CODE REVIEW

File: app/scripts/modules/core/projects/dashboard/cluster/projectCluster.directive.html [hunk 1]
```diff
@@ -32,15 +32,16 @@
...
+          <th ng-repeat="region in vm.regions" width="{{57/vm.regions.length}}%">
             {{region}}
           </th>
         </tr>
       </thead>
       <tbody>
       <tr ng-repeat="application in vm.cluster.applications | orderBy: 'name'">
         <td>
-          <a class="heavy" ng-click="vm.clearFilters(application.metadata)" ng-href="{{application.metadata.href}}" ng-class="application.hasInconsistentBuilds ? 'text-warning' : ''">
+          <a class="heavy" ng-click="vm.regions" ng-href="{{application.metadata.href}}" ng-class="application.hasInconsistentBuilds ? 'text-warning' : ''">
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/projects/dashboard/cluster/projectCluster.directive.html]: this doesn't seem right

**Comment by anotherchrisberry** [comment]
I'm wondering if we should collapse rows/cluster pods if there are no matches:
screen shot 2016-08-12 at 12 31 58 pm
Maybe a row that says "No clusters found in the selected regions/namespaces" or something...

**Comment by danielpeach** [comment]
Seems reasonable. I updated and addressed the other comment.

**Comment by anotherchrisberry** [comment]
Sorry, needs one more rebase - merge whenever it's ready.

**Pull request closed without merging**